woocommerce/woocommerce-paypal-payments
# DCC causes other gateways to disappear after checkout validation error (771)

When a validation error occurs on credit card payment, all gateways that are not PayPal and Credit card disappears.

### Steps to Test

1. Add a product to the cart and go to Checkout.
2. Fill checkout form fields but leave "Postcode / ZIP" empty 
3. Select "Credit card" payment, fill credit card fields and click "Place order".
4. After validation error appears, fill "Postcode / ZIP".

### What should happen
Same payment gateways should remain visible.

### What happened instead
All gateways that are not PayPal and Credit card disappears.
